docs/patroni_configuration.rst

```diff
@@ -41,6 +41,7 @@ PostgreSQL parameters controlled by Patroni
 Some of the PostgreSQL parameters **must hold the same values on the primary and the replicas**. For those, **values set either in the local patroni configuration files or via the environment variables take no effect**. To alter or set their values one must change the shared configuration in the DCS. Below is the actual list of such parameters together with the default values:
 
 - **max_connections**: 100
+  Note: In Patroni, the lower setting of max_connections is set to 25.
```

Comment:
Formatting with this change will look weird:
![image](https://github.com/user-attachments/assets/5b714085-5340-481d-ac67-adbd8b72868b)

Maybe just write everything on the same line...